### ISSUE TITLE
Package Type

### ISSUE THREAD

**Issue by baao** [tobias-redmann/dhl-php-sdk#2]
Hi,

```
        // FIXME: What is this
        $s['ShipmentItem']['PackageType'] = 'PL';
```

The 'PL' seems to be Palette - DHL doesn't seem to check for this on national shipments, but throws an error for international ones. If you change it to 'PK' it will work for national and international shipments.

Best regards!

**Comment by Petschko** [comment]
But what means "PK"? But thanks anyway for the info

**Comment by derpepe** [comment]
I think it stands for "package"!?

Peter D. notifications@github.com schrieb am Do. 15. Sep. 2016 um 14:31:

**Comment by baao** [comment]
Yes, it's package. However, it seems like DHL has removed that in the new API version

**Comment by Petschko** [comment]
Ah nice to know, thank you =)
